.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
This file can be included into your .gitignore to stop files like this from being added to your github repo.